metric_learn/sdml.py

```diff
@@ -73,11 +109,19 @@ def _fit(self, pairs, y):
                                     type_of_inputs='tuples')
 
     # set up (the inverse of) the prior M
-    if self.use_cov:
-      X = np.vstack({tuple(row) for row in pairs.reshape(-1, pairs.shape[2])})
-      prior_inv = np.atleast_2d(np.cov(X, rowvar=False))
-    else:
-      prior_inv = np.identity(pairs.shape[2])
+    # if the prior is the default (identity), we raise a warning just in case
+    if self.prior == 'identity':
+      msg = ("Warning, as of version 0.5.0, the default prior is now "
+             "'identity', instead of 'covariance'. If you still want to use "
+             "the inverse of the covariance matrix as a prior, "
+             "set 'prior'=='covariance' (it was the default in previous "
```

Comment:
the content of the parenthesis "it was the default..." is redundant, you can remove it

Reply:
that's right, done